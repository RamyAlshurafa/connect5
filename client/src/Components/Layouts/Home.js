import React, { Component } from "react";

import Navbar from "../CommonComponents/Navbar/index";

class Home extends Component {
  render() {
    return (
      <React.Fragment>
        <div className="home">Home</div>
        <Navbar />
      </React.Fragment>
    );
  }
}

export default Home;
