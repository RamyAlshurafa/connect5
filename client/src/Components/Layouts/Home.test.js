import React from 'react';
import Home from './Home';
import renderer from 'react-test-renderer';

test('Basic test for Home layout', () => {
  const component = renderer.create(
    <Home />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});