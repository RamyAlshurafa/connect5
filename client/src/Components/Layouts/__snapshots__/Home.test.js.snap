// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic test for Home layout 1`] = `
<div
  className="home"
>
  Home
</div>
`;
